account_invoice_import/models/account_invoice.py

```diff
@@ -0,0 +1,30 @@
+# © 2015-2016 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import models, api, _
+
+
+class AccountInvoice(models.Model):
+    _inherit = 'account.invoice'
+
+    @api.multi
+    def name_get(self):
+        """Add amount_untaxed in name_get of invoices"""
+        res = super(AccountInvoice, self).name_get()
+        if self._context.get('invoice_show_amount'):
+            new_res = []
+            for (inv_id, name) in res:
+                inv = self.browse(inv_id)
+                display_currency = inv.currency_id
```

Comment:
Add line


```
                pre = post = u''

```
to avoid this error:

```
Error:
Odoo Server Error

Traceback (most recent call last):
  File "/opt/odoo/odoo-11.0/odoo/http.py", line 647, in _handle_exception
    return super(JsonRequest, self)._handle_exception(exception)
  File "/opt/odoo/odoo-11.0/odoo/http.py", line 307, in _handle_exception
    raise pycompat.reraise(type(exception), exception, sys.exc_info()[2])
  File "/opt/odoo/odoo-11.0/odoo/tools/pycompat.py", line 87, in reraise
    raise value
  File "/opt/odoo/odoo-11.0/odoo/http.py", line 689, in dispatch
    result = self._call_function(**self.params)
  File "/opt/odoo/odoo-11.0/odoo/http.py", line 339, in _call_function
    return checked_call(self.db, *args, **kwargs)
  File "/opt/odoo/odoo-11.0/odoo/service/model.py", line 97, in wrapper
    return f(dbname, *args, **kwargs)
  File "/opt/odoo/odoo-11.0/odoo/http.py", line 332, in checked_call
    result = self.endpoint(*a, **kw)
  File "/opt/odoo/odoo-11.0/odoo/http.py", line 933, in __call__
    return self.method(*args, **kw)
  File "/opt/odoo/odoo-11.0/odoo/http.py", line 512, in response_wrap
    response = f(*args, **kw)
  File "/opt/odoo/odoo-server/addons/web/controllers/main.py", line 930, in call_kw
    return self._call_kw(model, method, args, kwargs)
  File "/opt/odoo/odoo-server/addons/web/controllers/main.py", line 922, in _call_kw
    return call_kw(request.env[model], method, args, kwargs)
  File "/opt/odoo/odoo-11.0/odoo/api.py", line 689, in call_kw
    return call_kw_multi(method, model, args, kwargs)
  File "/opt/odoo/odoo-11.0/odoo/api.py", line 680, in call_kw_multi
    result = method(recs, *args, **kwargs)
  File "/opt/odoo/custom/addons/account_invoice_import/models/account_invoice.py", line 26, in name_get
    inv.amount_untaxed, pre=pre, post=post)
UnboundLocalError: local variable 'post' referenced before assignment

```

Reply:
In Python 3 every string is unicode, so no need for the `u`

Reply:
ok will do this. @astirpe any idea to improve coverage ?

Reply:
Thanks!
I cannot check the coverage, as one test is failing...

Reply:
I create a merge request for failing test because it is not on this module

here it is : #53

Reply:
The line you added `pre = post = ''` is in the wrong place. It should be placed inside the for loop.

Reply:
done

Reply:
Thanks. Now the other line, the one outside the loop (`pre = post = ''`), can be removed.

Reply:
I mean line 13